packages/veritone-react-common/src/helpers/withMuiThemeProvider.js

```diff
@@ -1,48 +1,68 @@
 import React from 'react';
-import { func } from 'prop-types';
-import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
-import blue from '@material-ui/core/colors/blue';
-
-export default function withMuiThemeProvider(Component) {
-  class WrappedWithMuiTheme extends React.Component {
-    static displayName = Component.displayName || Component.name;
-    static propTypes = {
-      forwardedRef: func
-    };
-
-    render() {
-      const { forwardedRef, ...rest } = this.props;
-      return (
-        <MuiThemeProvider
-          theme={createMuiTheme({
-            palette: {
-              primary: {
-                light: blue[300],
-                main: blue[500],
-                dark: blue[700]
-              }
-            },
-            typography: {
-              button: {
-                fontWeight: 400
-              }
-            }
-          })}
-        >
-          <Component ref={forwardedRef} {...rest} />
-        </MuiThemeProvider>
-      );
-    }
+import { oneOfType, func, node, objectOf, object, any } from 'prop-types';
+import {
+  MuiThemeProvider,
+  createMuiTheme,
+  jssPreset
+} from '@material-ui/core/styles';
+import { create } from 'jss';
+import JssProvider from 'react-jss/lib/JssProvider';
+
+import { guid } from './guid';
+
+export class VSDKStyleWrapper extends React.PureComponent {
```

Comment:
believe PureComponent isn't suitable for a top level component because it'll prevent children from updating properly: https://reactjs.org/docs/react-api.html#reactpurecomponent

> Furthermore, React.PureComponent’s shouldComponentUpdate() skips prop updates for the whole component subtree. Make sure all the children components are also “pure”.